tips-and-workarounds.md

```diff
@@ -49,36 +48,28 @@ on:
   pull_request:
     types:
       - closed
-  workflow_dispatch:
 
 jobs:
   cleanup:
     runs-on: ubuntu-latest
-    permissions:
```

Comment:
Can we add this section back? I'd still recommend users explicitly define what `permissions` a workflow needs